lib/ddtrace/contrib/action_mailer/events/deliver.rb

```diff
@@ -34,6 +35,8 @@ def process(span, event, _id, payload)
             span.set_tag(Ext::TAG_MAILER, payload[:mailer])
             span.set_tag(Ext::TAG_MSG_ID, payload[:message_id])
 
```

Comment:
Minor: ⬆️ The comment on line 10 is incorrect (it's actually the deliver.action_mailer event).